src/Stratis.Bitcoin.Tests.Common/TestBase.cs

```diff
@@ -86,7 +86,7 @@ public static string GetTestDirectoryPath(object caller, [System.Runtime.Compile
         /// <returns>The path of the directory.</returns>
         public static string GetTestDirectoryPath(string testDirectory)
         {
-            return Path.Combine("TestCase", testDirectory);
+            return Path.Combine(@"..\..\..\..\TestCase", testDirectory);
```

Comment:
Where is that leading to? `src`? Can you explain why is this good?

Reply:
The Mac and Linux builds have trouble with this path I think.
Maybe try 
```suggestion
            return Path.Combine("..", "..", "..", "..", "TestCase", testDirectory);
```

Reply:
Fixed